BD-1358: Update Message Steps doc GA (#3519)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/canvas/canvas_components/action_paths.md

In the precedeing example, if **Ranking** is off and when a user performs one of the trigger events ("Start Session" or "Make Purchase"), that user is advanced in the relevant action group based on the trigger action.

> Note that Canvas entry properties differ from event properties. Canvas entry properties are properties from the event that kicked off the Canvas. These properties can only be used in the first full step of a Canvas. On the other hand, event properties originate from an event or action that occurs as the user goes through their workflow.

Reviewer: Suggestions to remove slang:

```suggestion
Note that Canvas entry properties differ from event properties. Canvas entry properties are properties from the event that started the Canvas. These properties can only be used in the first full step of a Canvas. Conversely, event properties originate from an event or action that occurs as the user goes through their workflow.
```

Reviewer: If "started" is weird here, potentially "initiated" or "triggered"? 🤔  Like "...the event that initially triggered the Canvas"
